mm2src/docker_tests/qrc20_tests.rs

```diff
@@ -367,8 +404,18 @@ fn qrc20_coin_conf_item(ticker: &str) -> Json {
 fn trade_base_rel((base, rel): (&str, &str)) {
     /// Generate a wallet with the random private key and fill the wallet with Qtum (required by gas_fee) and specified in `ticker` coin.
     fn generate_and_fill_priv_key(ticker: &str) -> [u8; 32] {
-        let priv_key = SecretKey::random(&mut rand4::thread_rng()).serialize();
+        if ticker == "QTUM" {
+            //Segwit QTUM
+            wait_for_estimate_smart_fee(30).expect("!wait_for_estimate_smart_fee");
+            let qtum_balance = MmNumber::from("10").to_decimal();
```

Comment:
10.into() that is also used below will be way shorter than `MmNumber::from().to_decimal().clone`.